src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -360,6 +379,18 @@ public static Validity oneYear() {
             return new Validity(start, end);
         }
 
+        /**
+         * Creates a validity duration starting one hour in the past and ends after two days.
+         */
+        public static Validity twoDays() {
```

Comment:
> It may even reduce the need for explicit abstractions like this.

I agree! I removed the Validity object entirely and used start and end OffsetDateTime objects instead